packages/docs/components/SideNav/SideNav.tsx

```diff
@@ -39,7 +40,7 @@ export const SideNav: React.FC = () => {
             Colors
           </SideNavLink>
           <SideNavLink href="/Icons/IconsPage" as="/icons">
-            Icons
+            Icon
```

Comment:
```suggestion
            Icons
```

Reply:
The name of the component is "Icon" right? I have been trying to align these to the component name

Reply:
Hmm... it seems a little weird referring to all the icons as just `Icon`.